mmv1/third_party/terraform/services/activedirectory/resource_active_directory_domain_trust_test.go

```diff
@@ -62,6 +62,7 @@ resource "google_active_directory_domain_trust" "ad-domain-trust" {
     trust_direction         = "OUTBOUND"
     trust_type              = "FOREST"
     trust_handshake_secret  = "Testing1!"
+	deletion_protection = false
```

Comment:
Spacing is a little wrong here. Maybe tabs were used

Reply:
Updated. Thanks!